LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Jacob Bolda
+Copyright (c) 2020 Jacob Bolda
```

Comment:
```suggestion
Copyright (c) 2018, 2019, 2020 Jacob Bolda
```
We want to keep the original copyright year and update it for every year that we made a material change. (Which happens to be all of them.)